### PR TITLE
Sukruthbhat/vz 7157

### PR DESCRIPTION
**Pull request by SukruthBhat**
Updated development.md by comparing it with the updated Makefile.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -81,12 +81,23 @@ kubectl apply -f k8s/manifests/verrazzano-monitoring-operator.yaml
...
 To run unit tests:
 
 ```
 make unit-test
 ```
 
+To perform both static analysis and run unit test in one go:
+
+```
+make check
```
Comment (gilbode) [inline docs/development.md:98]: Last I checked this only runs the linters though the behavior described here sounds better

Comment (SukruthBhat) [reply docs/development.md:98]: Should I remove it then??

Comment (SukruthBhat) [reply docs/development.md:98]: .PHONY: check
check: golangci-lint unit-test
Target command check was calling both the linter target command and unit test command..that's why I used that description.

Comment (gilbode) [reply docs/development.md:98]: My mistake - thought this was verrazzano repo not verrazzano-monitoring-operator.

**Review by gilbode — APPROVED** [review]

**Pull request merged**